access/types/forms.py

```diff
@@ -452,7 +452,7 @@ def grade(self):
 
         return (points, error_groups, error_fields)
 
-    def compare_values(self, method, val, cmp):
+    def compare_values(self, method, val, cmp, float_rel_tol=1e-09, float_abs_tol=0.0):
```

Comment:
Because the old code used 0.02, shouldn't the default also be 0.02, to maintain backwards compatibility?